SETUP/lint_charsuites.php

```diff
@@ -19,6 +20,24 @@
         exit(1);
     }
 
+    // Validate that the character suite does not contain codepoints we convert
+    // to ASCII
+    $disallowed_characters = convert_codepoint_ranges_to_characters(get_disallowed_codepoints());
+    $charsuite_characters = convert_codepoint_ranges_to_characters($charsuite->codepoints);
+    $characters = array_intersect($disallowed_characters, $charsuite_characters);
+    if($characters != [])
+    {
+        echo "ERROR: disallowed characters found in suite\n";
+        foreach($characters as $char)
+        {
+            echo sprintf("    %s: %s\n", $char, utf8_chr_to_hex($char));
```

Comment:
should we use \t here?

Reply:
Sorry, use `\t` where in the string?

Reply:
I meant for alignment. Instead of using 4 spaces before %s, use a \t character. Nothing critical. Just curious.

Reply:
My understanding is that using `\t` can be somewhat fraught, because it means different things in different places, depending on where tab stops are set.

Another question, though -- should it be `%1$s` and `%2$s`, or is that irrelevant because there's no translation to worry about?

Reply:
> Another question, though -- should it be `%1$s` and `%2$s`, or is that irrelevant because there's no translation to worry about?

It's irrelevant because it isn't translated.